packages/apollo-react-relay-duct-tape/src/storeObservation/compiledHooks/useCompiledLazyLoadQuery.ts

```diff
@@ -109,7 +109,7 @@ export function useCompiledLazyLoadQuery(
   const { data } = useApolloQuery(watchQueryDocument, {
     client,
     variables,
-    fetchPolicy: "cache-only",
```

Comment:
Updated it now, still not sure why we would not need it in watchQuery. Could you explain?

Reply:
I will try but probably @alloy would do better job. Nonetheless:
* `watchQueryDocument` is used to subscribe to cache updates of fields selected in graphql document - this is the node query against the cache and should always have `cache-only` policy because of that
* `executionQueryDocument` is used to execute real query against the query fields you directly specified in your query which fills in cache with everything you have in your entire query tree

So we still need to change back to have `cache-only` on watch query

Reply:
To illustrate it a bit better, I added a couple of properties in people app demo and made sure to generate with debug comments. I reorder things so that execution and watch query are next to each other:

![image](https://github.com/microsoft/graphitation/assets/19153565/c2e38ebe-48e5-4c86-bed5-56e3e8e2e952)

Reply:
But the executionQuery was using "network-only" anyway - which is what we want. So, how are we affecting anything by this change?